parts/k8s/addons/kubernetesmasteraddons-antrea-daemonset.yaml

```diff
@@ -0,0 +1,495 @@
+apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
Some backstory: the `parts/k8s/addons/` directory is a "legacy" addons implementation. This is not obvious. :/

So, let's move this spec under `parts/k8s/containeraddons/`, which will allow us to enable much more flexible user configuration.

Reply:
Ok.. will move the YAML file under containeraddons dir.  Thanks